src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -762,14 +703,17 @@ private Uri PrepareUri(Uri uri)
 
         private static Uri CheckProtocol(Uri uri)
         {
-            if (uri == null) { throw new ArgumentNullException(nameof(uri)); }
+            if (uri == null) 
+            { 
+                throw new ArgumentNullException(nameof(uri)); 
+            }
```

Comment:
Not for the PR (we could update all code base) - we could use new `ArgumentNullException.ThrowIfNull` and such others.